### PR TITLE
various changes

### PR DESCRIPTION
**Pull request by elina-holmes**

### CODE REVIEW

File: classify.py [hunk 1]
```diff
@@ -329,120 +340,118 @@ def evaluate_samesource(desc, dataset, n_frequent_words, tokens_per_sample, prep
...
-    exp.addSearch('classifier', [('bray_svm', svc_br), ('man_svm', svc_ma),
+    # exp.parameter('classifier', ('dist_man', dist_ma))
+    exp.parameter('classifier', ('bray_logit', logit_br))
+    exp.addSearch('classifier', [('dist_man', dist_ma), ('dist_bray', dist_br), ('man_logit', logit),
                                  ('bray_logit', logit_br)], include_default=False)
-    # exp.addSearch('classifier', [('dist', dist), ('svc', svc)], include_default=False)
 
     exp.parameter('calibrator', lir.ScalingCalibrator(lir.KDECalibrator()))
-    exp.parameter('repeats', 10)
+    exp.parameter('repeats', 2)
```
Comment (wowtor) [inline classify.py]: is this change intended?

File: classify.py [hunk 2]
```diff
@@ -329,120 +340,118 @@ def evaluate_samesource(desc, dataset, n_frequent_words, tokens_per_sample, prep
     recall = np.round(np.mean(lrs[y_all == 1] > 1), 3)
     precision = np.round(np.mean(y_all[lrs > 1] == 1), 3)
 
-    return cllr, cllrmin, cllrcal, acc, recall, precision
+    return cllr, cllrmin, cllrcal, acc, recall, precision, lrs, y_all
```
Comment (wowtor) [inline classify.py]: you may consider using namedtuple:
`Metrics = collections.namedtuple('Metrics', ['cllr', 'cllrmin', 'cllrcal', ...])`

And then: `return Metrics(cllr, cllrmin, ...)`

In `aggregate_results` you can refer to it in several ways, like `results.cllr` or `results._asdict()`

**Review by wowtor — APPROVED** [review]

File: classify.py (hunk 1 above)
Comment (elina-holmes) [reply classify.py]: no

File: classify.py (hunk 2 above)
Comment (elina-holmes) [reply classify.py]: done
